tests/Unit/Training/WaitingListTest.php

```diff
@@ -0,0 +1,182 @@
+<?php
+
+namespace Tests\Unit\Training;
+
+use App\Models\Mship\Account;
+use App\Models\Mship\Role;
+use App\Models\Training\WaitingList;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use Tests\TestCase;
+
+class WaitingListTest extends TestCase
+{
+    use DatabaseTransactions;
+
+    private $waitingList;
+    private $staffUser;
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->waitingList = factory(WaitingList::class)->create();
+
+        $this->staffUser = factory(Account::class)->create();
+
+        $this->staffUser->roles()->attach(Role::find(1));
+    }
+
+    /** @test * */
+    public function itHasASlugRouteKey()
+    {
+        $this->assertEquals('slug', $this->waitingList->getRouteKeyName());
+    }
+
+    /** @test * */
+    public function itHasAName()
+    {
+        $this->assertNotNull($this->waitingList->name);
+        $this->assertNotNull($this->waitingList->slug);
+    }
+
+    /** @test * */
+    public function itCanHaveStudents()
```

Comment:
You hinted at test name standardisation in another PR; I totally agree. Can we decide on what we will call people and then we will look to update existing tests in another PR? Students/Members/Users

Reply:
Generally, I think when testing we should use the name on the user story so that we can assert in our own minds that we are actually meeting the requirements. Thoughts on that?

Reply:
Yep, that makes sense. We should ensure it is consistent throughout the test/feature in that case